ophyd/areadetector/filestore_mixins.py

```diff
@@ -92,15 +100,13 @@ def stage(self):
 
         # AD does this same templating in C, but we can't access it
         # so we do it redundantly here in Python.
-        fn = self.file_template.get() % (full_read_path, self._filename,
-                                         self.file_number.get())
+        self._fn = self.file_template.get() % (full_read_path,
+                                               self._filename,
+                                               self.file_number.get())
 
         if not self.file_path_exists.get():
```

Comment:
Am I correct in recalling that directory creation will not be touched on in this file (purposefully)?